testing/opensmtpd-extras/remove-decls.patch

```diff
@@ -0,0 +1,15 @@
+--- opensmtpd-extras-201703132115/extras/tables/table-ldap/ber.h
++++ opensmtpd-extras-201703132115-patched/extras/tables/table-ldap/ber.h
```

Comment:
Please replace `opensmtpd-extras-201703132115` with `a` and `opensmtpd-extras-201703132115-patched` with `b` and write a description why is this patch here.